postgres/docker-entrypoint.sh

```diff
@@ -0,0 +1,340 @@
+#!/usr/bin/env bash
```

Comment:
This is copy/pasted from here https://github.com/docker-library/postgres/blob/master/docker-entrypoint.sh